kubernetes/table_kubernetes_pod.go

```diff
@@ -618,23 +618,23 @@ func transformPodCpuAndMemoryUnit(ctx context.Context, d *transform.TransformDat
 
 	if param == "limit" {
 		limitCPUMemoryMaps := make([]map[string]interface{}, 0)
-	
-		limitCPUMemoryMap := make(map[string]interface{})
-	
+
 		for _, container := range containers {
+			// Create a new map for each container
+			limitCPUMemoryMap := make(map[string]interface{})
```

Comment:
@ParthaI, what is the advantage of creating a new map for each container? How does it make the code more effective?